### ISSUE TITLE
Accessibility fixes

### ISSUE THREAD

**Issue by stephaniesimms** [CDLUC3/dmptool#89]
Test scheduled for Apr 2.
https://drive.google.com/file/d/11Hq_quzgNq8YL-vneI-4-d_NrmCClUa0/view

**Basic Accessibility Check of DMPTool Prototype1**

For basic accessibility, compliance with WCAG 2.0 Level AA requires that all page functionality, including links, buttons, and form controls, is available using only the keyboard. In addition, Level AA requires a color contrast ratio of 4.5:1 for normal text and 3:1 for large text.

To conduct the test, we began at browser's address bar and used only the TAB, ENTER, SPACEBAR, and arrow keys to navigate the page. We also used the Chrome WAVE Evaluation Tool to perform automated accessibility testing. Our findings are presented below.

**Access all interactive elements of page – Failed**
I was able to access all interactive elements on the homepage and My Dashboard using only my keyboard. Tabbing proceeds in a logical order. Navigation menus are keyboard accessible. (Note: There are sort links in the table that are not visible, but are accessible via keyboard.)

However, on the Write Plan tab, I was unable to access sections individually. In order to access any of the sections, I had to expand all of them. In addition, I could not access the buttons in the text editor.

**Obvious keyboard focus – Failed**
It was difficult to identify keyboard focus for some interactive elements of the homepage and My Dashboard. The highlight style has been applied inconsistently. Blue highlights around blue buttons are extremely difficult to see. On the Write Plan tab, it is very difficult to follow the focus.

**Skip navigation/Skip to main content link – Failed**
A "Skip to content" link is not present.
Proper heading structure – Failed
The homepage skips heading levels.

**Zoom 200% Check – Failed**
Layout breaks at 200% magnification. Some page elements are not visible.

**WAVE Findings**
1. There are missing form labels.
2. There are empty links and buttons.
3. There are broken ARIA references on the Write Plan tab.

**Comment by stephaniesimms** [comment]
replaced w/individual issues in roadmap and dmptool repos based on accessibility report received in May 18
